dlt/sources/rest_api/__init__.py

```diff
@@ -63,7 +63,31 @@
 ]
 
 
-@decorators.source
+@decorators.source(
+    example_values={
+        "client": {
+            "base_url": "https://jaffle-shop.scalevector.ai/api/v1/",
+            "paginator": "header_link"
+            },
+        "resources": ["customers", "orders"],
+        # or
+        # "resources": [
```

Comment:
I don't see a nice solution way of putting this into the config.toml file.
there also are plenty of other even longer examples in the comment below, so there is some duplication.

maybe we need a dlthub just for rest-api configs